spec/models/measure_spec.rb

```diff
@@ -30,14 +30,14 @@
       expect(subject.errors[:parent_id]).to(include("can't be the same as id"))
     end
 
-    it "can't be set to a measure with :measuretype.has_parent = false" do
-      subject.parent_id = described_class.create(
-        measuretype: FactoryBot.create(:measuretype, :parent_not_allowed),
-        title: "no parent"
-      ).id
-      expect(subject).to be_invalid
-      expect(subject.errors[:parent_id]).to(include("is not allowed for this measuretype"))
-    end
+    # it "can't be set if measure's measuretype :measuretype.has_parent = false" do
```

Comment:
@parndt: could you update this test? apparently the previous check was checking if a measure's parent can have parents not the measure itself (set in the measuretype) - previously this has not been an issue, as we only allowed measures to have parents of the same measuretype